### PR TITLE
CIAM-2979 RH-SSO Containers - prepare for 7.6

### PR DESCRIPTION
**Pull request by drichtarik**
@iankko
I did test builds in cpaas and our custom pipeline as well. Both passed. I roughly checked the scratch image builds. They did start without and error and startup log contained RH-SSO 7.6.0 version

### CODE REVIEW

**Review by iankko — APPROVED** [review]
TL;DR: LGTM AFAICT (see below for additional notes) @drichtarik Thanks, Denis!

Longer version - confirmed that:

- All instances of `7\.5' string were replaced with their '7\.6' counterparts (there are three left in comments, but that's expected / OK),
- All instances of `sso75` were replaced with their `sso76` counterparts,
- There's no file named `sso75` in the repo (so looks all templates & docs files were renamed to their 76 counterparts):
```
[iankko@localhost redhat-sso-7-openshift-image]$ find . -name sso75* | wc -l
0
```
- MD5 sum of the `keycloak-server-overlay.zip` matches the expected value,
- Versions of both `WILDFLY_VERSION` and `GALLEON_WILDFLY_VERSION` used in the repo match their respective versions, as present in latest JBoss EAP 7.4 with OpenJDK11 container image:
```
$ docker run -it registry.redhat.io/jboss-eap-7/eap74-openjdk11-openshift-rhel8:latest env | grep WILDFLY_VERSION
WILDFLY_VERSION=7.4.4.GA-redhat-00011
GALLEON_WILDFLY_VERSION=5.1.2.Final
```

Based on the above, looks the PR is correct (all necessary updates were performed). ACK from me.
